### ISSUE TITLE
Integration with `Open URL in container` extension

### ISSUE THREAD

**Issue by maximbaz** [ninevra/Simple-Temporary-Containers#15]
Hello,

Thanks for the wonderful extension!

I'm trying to build an integration with [Open URL in container](https://addons.mozilla.org/en-US/firefox/addon/open-url-in-container/) extension, such that I use shell script to open a new link in a temporary container that will be managed by your extension.

From the first glance, it doesn't seem possible to pass any other "custom" information via "Open URL in container", that can be used by this extension to add the tab to the database. 

I was thinking that we could perhaps use a naming convention? For example, if a new container gets created with a name `Temp <8 chars>`, treat it as a temporary container as if created by this extension.

Basically, I want to script my way into creating a temporary session that I can start with a shell, and have this container destroyed when the last tab is closed.

What do you think?
